Node 6.2.0 release Testnet (#928)
Concordium/concordium.github.io: source/mainnet/net/release-notes/release-notes-lp.rst

Testnet
-------

    November 28, 2023

>     Version 6.2.3 deprecates the gRPC v1 API. This removes the configuration options ``CONCORDIUM_NODE_RPC_SERVER_PORT``, ``CONCORDIUM_NODE_RPC_SERVER_ADDRESS``, ``CONCORDIUM_NODE_RPC_SERVER_TOKEN``, ``CONCORDIUM_NODE_DISABLE_RPC_SERVER_NODE_ENDPOINTS`` and their command line equivalents. An additional health-check service was also added to the V2 GRPC API. This service conforms to the `standard GRPC health service API <https://github.com/grpc/grpc-proto/blob/master/grpc/health/v1/health.proto>`__.

Reviewer: ```suggestion
    Version 6.2.3 removes the V1 gRPC API. This removes the configuration options ``CONCORDIUM_NODE_RPC_SERVER_PORT``, ``CONCORDIUM_NODE_RPC_SERVER_ADDRESS``, ``CONCORDIUM_NODE_RPC_SERVER_TOKEN``, ``CONCORDIUM_NODE_DISABLE_RPC_SERVER_NODE_ENDPOINTS`` and their command line equivalents. An additional health-check service was also added to the V2 GRPC API. This service conforms to the `standard GRPC health service API <https://github.com/grpc/grpc-proto/blob/master/grpc/health/v1/health.proto>`__.
```

>     As part of the tokenomics changes, new endpoints have been added for validators. The existing endpoints will continue to work.

Reviewer: ```suggestion
    As part of the tokenomics changes the node has new configuration options that uses the new terminology. The existing options using the legacy terminology are still supported, however they are hidden.
```

    A ``DryRun`` endpoint has also been that allows simulating the execution of transactions.

>     The account map is now kept solely on disk in a separate lmdb database and it is no longer part of the internal block state database. This change results in significantly reduced resource usage for the node.

Reviewer: ```suggestion
    The account map is now kept solely on disk in a separate LMDB database and it is no longer part of the internal block state database. This change results in significantly reduced resource usage for the node.
```
